src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -434,6 +435,10 @@ private function registerProfilerConfiguration(array $config, ContainerBuilder $
             $loader->load('form_debug.xml');
         }
 
+        if ($this->validatorConfigEnabled) {
```

Comment:
simpler solution is to add a conflict rule in FrameworkBundle to forbid using an older version of symfony/validator when using FrameworkBundle 3.4